File: bridge-servlet/core/src/main/java/it/unibg/driveaura/bridge/core/HealthResponse.java
package it.unibg.driveaura.bridge.core;

/** Servlet-API-independent representation of the bridge health contract. */
public final class HealthResponse {
    public static final String API_VERSION = "1.0";
    public static final String SERVICE = "bridge-servlet";

    private HealthResponse() {
    }

    public static String json() {
        return "{\"apiVersion\":\"" + API_VERSION
                + "\",\"service\":\"" + SERVICE
                + "\",\"status\":\"ok\"}";
    }
}

File: remote-php/src/HealthResponse.php
<?php

declare(strict_types=1);

namespace DriveAura\Remote;

/** Builds the public health representation shared by the PHP entry point. */
final class HealthResponse
{
    public const API_VERSION = '1.0';
    public const SERVICE = 'remote-php';

    /** @return array{apiVersion: string, service: string, status: string} */
    public static function body(): array
    {
        return [
            'apiVersion' => self::API_VERSION,
            'service' => self::SERVICE,
            'status' => 'ok',
        ];
    }
}
